storage/oasis/nodeapi/history/runtime.go

```diff
@@ -43,6 +43,15 @@ func NewHistoryRuntimeApiLite(ctx context.Context, history *config.History, sdkP
 	}, nil
 }
 
+func (rc *HistoryRuntimeApiLite) Close() error {
+	for _, api := range rc.APIs {
+		if err := api.Close(); err != nil {
+			return err
```

Comment:
should this have the same logic as consensus to save the `firstErr` and continue attempting to close the other APIs first?

Reply:
good catch, thank you! implemented.